examples/getting-started-movielens/02-ETL-with-NVTabular.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "INPUT_DATA_DIR", os.path.expanduser("~/nvt-examples/movielens/data/")
</pre><p>Is it possible to not have this path as root? is there a specific reason we set this as root? I know this is from existing examples, but I really dont like that this is in root :)  can we set it as <code>/workspace/nvt-examples/...</code>  if it wont create too much work? what do you think? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
changed the path as advised

Reply:
@radekosmulski was changing path created any issue with the unit tests? let's be sure it wont :)